### PR TITLE
Refactor project

### PR DESCRIPTION
**Pull request by JoshMallery**
Let compare this to find a bug!

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ function generateRandomCover() {
   instantiateCover(randCover, randTitle, randDescriptor1, randDescriptor2);
```
Comment (kal-aalrajhi) [inline src/main.js]: Change to createCover

**Review by JoshMallery — COMMENTED** [review]
We need to adjust a variable name on line 45

**Review by JoshMallery — COMMENTED** [review]
great gif!

**Review by Wperricone — APPROVED** [review]
Love the extra work on the Form page!

**Review by kal-aalrajhi — APPROVED** [review]
Looks good!

**Review by JoshMallery — COMMENTED** [review]
Looking great

**Pull request merged**